docs/api_reference/dataframe.rst

```diff
@@ -3,9 +3,23 @@
 nisystemlink.clients.dataframe
 ======================
 
-.. automodule:: nisystemlink.clients.dataframe
-   :members:
-   :imported-members:
+.. autoclass:: nisystemlink.clients.dataframe.DataFrameClient
+   :exclude-members: __init__
```

Comment:
The ordering was getting messed up with this fix 😕
In `conf.py`, we set a sphinx autodoc option to always document the special member `__init__`. But it was last in the list after the client methods, which isn't expected. The way I fixed it was by manually excluding `__init__` from the autoclass and then specifying it as an automethod.